app/src/androidTest/java/org/mozilla/focus/activity/AccessAboutAndYourRightsPagesTest.kt

```diff
@@ -0,0 +1,129 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.activity
+
+import android.preference.PreferenceManager
+import android.support.annotation.IdRes
+import android.support.test.InstrumentationRegistry
+import android.support.test.espresso.IdlingRegistry
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.focus.R
+import org.mozilla.focus.helpers.SessionLoadedIdlingResource
+import org.mozilla.focus.utils.AppConstants
+
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.Espresso.pressBack
+import android.support.test.espresso.action.ViewActions.click
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
+import android.support.test.espresso.matcher.ViewMatchers.withId
+import android.support.test.espresso.matcher.ViewMatchers.withText
+import org.hamcrest.Matchers.containsString
+import org.mozilla.focus.fragment.FirstrunFragment.Companion.FIRSTRUN_PREF
+import org.mozilla.focus.helpers.EspressoHelper.openMenu
+
+// This test visits each page and checks whether some essential elements are being displayed
+// https://testrail.stage.mozaws.net/index.php?/cases/view/81664
+@RunWith(AndroidJUnit4::class)
+class AccessAboutAndYourRightsPagesTest {
+
+    @Rule
```

Comment:
I get error with Kotlin tests that states
`...runners.rules.ValidationError: The @Rule 'mActivityTestRule' must be public`
and test cannot start but after replacing @Rule with @get:Rule Kotlin tests can start and run.